resources/lang/fr/flights.php

```diff
@@ -17,18 +17,18 @@
     'arr'              => 'Arr',
     'level'            => 'Niveau',
     'alternateairport' => 'Aéroport Alternatif',
-    'inbound'          => 'Vols à l\'arrivée',
-    'outbound'         => 'Vols au départ',
-    'none'             => 'Auncun vols trouvé.',
+    'inbound'          => 'Vols entrants',
+    'outbound'         => 'Vols sortants',
+    'none'             => 'Ce vol n\'existe pas.',
```

Comment:
If you wanna say "No flights found." then it's "Aucuns vols trouvés." and if you wanna say "This flight doesn't exists" then it's "Ce vol n'existe pas."

Reply:
Ici je ne sais pas vraiment à quelle occasion c'est utilisé

Reply:
Alors le inbound/oubound c'est pour la page airport, ca affiche la liste des vols aux départs / à l'arrivée. Pour ca je ne sais pas trop s'il fallait changer. Ensuite pour le none c'est quand tu cherches un vol et qu'il en trouve pas, pour ca j'aurai laissé aucun vols trouvés